### ISSUE TITLE
Can I ask when will the new version release?

### ISSUE THREAD

**Issue by cybersecurity-research** [jtesta/ssh-mitm#48]
Hi jtesta

Two months ago. We talk about scp problem. You said new version will be release on dec 2021. I can't wait for it. Can I ask when will the new version release?  

Thanks!!!

**Comment by q2dg** [comment]
Last commit was from nearly one year ago...so it seems this project is pretty abandoned...

**Comment by jtesta** [comment]
> Last commit was from more than one year ago...so it seems this
> project is pretty abandoned...

No, I wouldn't say its abandoned.  I just happen to have my hands full
for longer than I thought.

I'd still be happy to merge PRs from the community, though!

**Comment by q2dg** [comment]
Thanks!
Just for the record...there's another project which seems similar to this one: https://github.com/ssh-mitm/ssh-mitm
Maybe it will help someone

**Comment by cybersecurity-research** [comment]
Got it! Thanks.

**Comment by hmaier1996** [comment]
> Last commit was from more than one year ago...so it seems this project is pretty abandoned...
> No, I wouldn't say its abandoned. I just happen to have my hands full for longer than I thought. I'd still be happy to merge PRs from the community, though!

**Do you have a roadmap for upcoming features?**

When do you plan to release a version with publickey and password authentication?

At the moment it's only possible to intercept publikey or password authentication but not both at the same time. Will this get fixed?

**Comment by hmaier1996** [comment]
The author of the other tool has contributed to this server some time ago.

@jtesta Is the other one based on this OpenSSH fork and what are the differences? Perhaps some features can be merged back in this project.
